### ISSUE TITLE
Bookdown publications

### ISSUE THREAD

**Issue by ckrusemd** [ekstroem/UEFA2020-prediction-competition#4]
Hi!
Would anyone be interested in publishing the predictions as a repo-specific Bookdown on Github Pages?
I have a template and a YAML file for this to happen. I'd be happy to render the current submissions and set it up to build automatically when a pull request is accepted.
Regards,
Christian

**Comment by ekstroem** [comment]
That sounds great. Is there anything I can do to help?

**Comment by ckrusemd** [comment]
I'll make a pull request with the files in them. There might be a part of Settings where you have to enable Pages for a specific branch and folder - I can send post the screenshots or you could grant me rights to do so (I will relinquish this power afterwards, as the bookdown should build automatically for new pull requests).

**Comment by ekstroem** [comment]
I think I might have set that up by now. If not then please let me know.

The page should be https://ekstroem.github.io/UEFA2020-prediction-competition/

**Comment by ckrusemd** [comment]
Thank you for approving the pull request. There is an issue with authentication when deploying. This will likely remedy the situation, if you can allocate the time. I'd be happy to take the helm as well.
https://docs.github.com/en/actions/reference/authentication-in-a-workflow

**Comment by ekstroem** [comment]
It's still making hiccups of some sort

**Comment by ckrusemd** [comment]
Right! I would need rights to access Settings to fix the key issue. Perhaps it's not worth the hassle. If so, you can disable the workflow in Actions or delete the .github folder.

**Comment by ekstroem** [comment]
Oh I thought I had given you those. If it's not working I'd be happy to click whereever you'd want me to
